xml/ha_storage_protection.xml

```diff
@@ -664,7 +664,18 @@ Timeout (msgwait)  : 10
    <procedure xml:id="pro-ha-storage-protect-sbd-config">
    <title>Editing the SBD configuration file</title>
     <step>
-     <para>Open the file <filename>/etc/sysconfig/sbd</filename>.</para>
+     <para>Open the file <filename>/etc/sysconfig/sbd</filename> and use
+      the following entries:</para>
+     <screen>SBD_PACEMAKER=yes
+SBD_STARTMODE=always
+SBD_DELAY_START=no
+SBD_WATCHDOG_DEV=/dev/watchdog
+SBD_WATCHDOG_TIMEOUT=5</screen>
```

Comment:
We could omit SBD_WATCHDOG_TIMEOUT setting here for the case of SBD with shared disk, given that the watchdog timeout setting is basically from the metadata on the shared sbd disk.